include/queryfunctions.php

```diff
@@ -73,6 +73,74 @@ function GetMovementsForWorkout($workoutId){// adding join to pull movementInsta
         return $movements;
     }
 
+    function GetAllInstances(){ // highkey over kill on rows rn
+        $instances = dbQuery("
+        SELECT * FROM movementInstances INNER JOIN sets 
+        ON movementInstances.instanceId = sets.instanceId
+        INNER JOIN workouts 
+        ON sets.workoutId = workouts.workoutId
+        WHERE isProgram IS NULL
+        ORDER BY dateTimeStarted DESC
+        ")->fetchAll();
+        return $instances;
+    }
+
+    //volume functions
+
+
+    function GetVolume(){ //did stuck in the for loop and it worked, very scared to touch it now :,)
+        $instances = dbQuery("
+        SELECT movementInstances.instanceId,movementId, reps, weight, dateTimeStarted
+        FROM movementInstances INNER JOIN sets 
+        ON movementInstances.instanceId = sets.instanceId
+        INNER JOIN workouts 
+        ON sets.workoutId = workouts.workoutId
+        WHERE isProgram IS NULL
+        ORDER BY movementId, dateTimeStarted 
+        ")->fetchAll(); //isComplete needs to be taken into account later and userId should be specified
+
+        $volume = [];
+        $vol = 0;
+        $id = 0; // id is initialized to 0
+        foreach($instances as $i){ // organized by movement id as well as ordered by date (damn...)
+            
+            
+            if($id==$i['dateTimeStarted']){ // adding to the volume
+                $vol = $vol+($i['reps'] * $i['weight']);
```

Comment:
I think this is more readable
```suggestion
                $vol += ($i['reps'] * $i['weight']);
```